package.json

```diff
@@ -88,18 +88,28 @@
     "@typescript-eslint/eslint-plugin": "^5.9.1",
     "@typescript-eslint/parser": "^5.9.1",
     "apollo": "^2.33.9",
+    "assert": "^1.5.0",
+    "buffer": "^5.7.1",
+    "crypto-browserify": "^3.12.0",
     "eslint": "^7.27.0",
     "eslint-config-oceanprotocol": "^1.5.0",
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-react": "^7.28.0",
     "eslint-plugin-react-hooks": "^4.3.0",
     "file-loader": "^6.2.0",
+    "https-browserify": "^1.0.0",
     "husky": "^7.0.4",
+    "os-browserify": "^0.3.0",
     "prettier": "^2.5.1",
     "pretty-quick": "^3.1.3",
+    "process": "^0.11.10",
+    "react-app-rewired": "^2.1.11",
```

Comment:
do we really need all those new packages? Like, `assert`, `process`, `react-app-rewired`, `url` can all be removed from here until we really need them when used in next.config.js

Reply:
`react-app-rewired` was due to copy paste frenzy. The rest are just devDependecies, i could try removing them but this whole mess was very time consuming